Automattic/wp-calypso
# Pricing page rework: Add Recommendations section

#### Proposed Changes

* This PR adds the recommendations section to the pricing page along with the company logos

#### Testing Instructions





* Boot up this PR 
    * Run `git fetch && git checkout add/jetpack-pricing-recommendations`
    * Run `yarn start-jetpack-cloud`
    * Goto [http://jetpack.cloud.localhost:3001/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3001/pricing?flags=jetpack/pricing-page-rework-v1)
* Or click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* Confirm that you see the recommendations section along with the company logos
* Confirm that the design matches the Figma design

For mobile view, please compare with the same section on [jetpack.com](https://jetpack.com/)

#### Pre-merge Checklist






Completes 1202796695664022-as-1202796695664064/f
Figma: m0y1jHTuDpvMPkFsOJBoNP-fi-3715%3A132855


<img width="1202" alt="image" src="https://user-images.githubusercontent.com/18226415/186156476-0f00f4a6-6d5f-4c44-bbb4-528ed2bb6638.png">
<img width="297" alt="image" src="https://user-images.githubusercontent.com/18226415/186156602-460a90e9-a06b-4d14-9e7c-e24650f90cdc.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202796695664064
